src/ui/styles/components/compact-row.css

```
@@ -0,0 +1,65 @@
+.compact-row--ready:hover {
+  color: var(--text-secondary);
+}
+
+.compact-row--ready:hover .compact-chevron {
+  color: var(--text-secondary);
+}
+
+.compact-chevron {
+  width: 14px;
+  height: 14px;
+  fill: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)